iogt_users/tests.py

```diff
@@ -30,3 +30,22 @@ def test_user_locked_out_without_filling_registration_survey_form(self):
     def test_anonymous_user_can_browse_public_urls(self):
         response = self.client.get(self.home_page.url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_admin_without_filling_registration_survey_form(self):
+        admin_user = AdminUserFactory(has_filled_registration_survey=False)
+
+        self.client.force_login(admin_user)
+        response = self.client.get(self.home_page.url)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_user_with_admin_access_do_not_need_to_fill_registration_survey_form(self):
+        admin_access_permission = Permission.objects.get(codename='access_admin')
+        group = GroupFactory()
+        group.permissions.add(admin_access_permission)
+
+        self.user.groups.add(group)
```

Comment:
I thought this might be the kind of code that would be in an 'admin user factory', so that if you need an admin user there is a standard way to do it across the whole test suite. I'm happy to leave this as it is if you think it's a premature optimisation.

Reply:
Admin user already has 'wagtail admin access'. But in this test-case we are giving wagtail admin access to a simple user, so I think it should be as it is here.

Reply:
"Superuser" has all permissions. "Admin user" has admin access permission. This is the distinction we're failing to make at the moment, but I think it might be useful. Let's leave it for now and see whether this comes up again in future.